src/qt/listbox.cpp

```diff
@@ -147,14 +147,15 @@ wxString wxListBox::GetString(unsigned int n) const
     return wxQtConvertString( item->text() );
 }
 
-void wxListBox::SetString(unsigned int n, const wxString& WXUNUSED(s))
+void wxListBox::SetString(unsigned int n, const wxString& s)
 {
     QListWidgetItem* item = m_qtListWidget->item(n);
     wxCHECK_RET(item != NULL, wxT("wrong listbox index") );
     if ( m_hasCheckBoxes )
     {
         item->setFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled | Qt::ItemIsSelectable);
         item->setCheckState(Qt::Unchecked);
+        item->setText(wxQtConvertString(s));
```

Comment:
Sorry if I'm missing something obvious but shouldn't this be outside of `m_hasCheckBoxes` test?

Reply:
You are absolutely correct, I have updated the change to do this outside of this check. Thanks!